src/load_secrets_from_vault/load.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import logging
+import os
+
+import load_secrets_from_vault.context
+
+from load_secrets_from_vault.load import load_secrets
+from lib.logging import setup_logging
```

Comment:
I had a look at that function and it looks like the format isn't actually being set 🤔

Reply:
oh my. [that is odd](https://github.com/artsy/opstools/blob/140a26e9d9598b55cf189abb3d248760d2549cc2/src/lib/logging.py#L24).

Reply:
Yep, good catch, format is defined but not actually set.

Question is should we set it or not. The logging actually looks better default. Using `ecr_check_repos_for_terraform` script for example, currently:

```
(opstools) artsy:opstools jxu$ python src/ecr_check_repos_for_terraform/check.py
INFO:root:Checking ECR for repositories that perhaps should be added to Terraform...
INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
INFO:root:Repositories found: ['conversations_purchase_predictor']
```

If we set the format:

```
(opstools) artsy:opstools jxu$ git diff
diff --git a/src/lib/logging.py b/src/lib/logging.py
index a8de146..cfb850f 100755
--- a/src/lib/logging.py
+++ b/src/lib/logging.py
@@ -22,6 +22,9 @@ def setup_logging(level):
   ''' configure root level logging '''
   logging.basicConfig(level=level)
   format = '%(levelname)s: %(message)s'
+  formatter = logging.Formatter(format)
+  for handler in logging.root.handlers:
+     handler.setFormatter(formatter)
```

logging becomes:

```
(opstools) artsy:opstools jxu$ python src/ecr_check_repos_for_terraform/check.py                                                           
INFO: Checking ECR for repositories that perhaps should be added to Terraform...
INFO: Found credentials in shared credentials file: ~/.aws/credentials
INFO: Repositories found: ['conversations_purchase_predictor']
```

Default format shows the module that spit out the log, which is good to have.

Reply:
so i go for removing format definition. done.

Reply:
oh that script to check for ECR repos that could be in terraform is fun!